src/types/types.ts

```diff
@@ -308,6 +308,7 @@ export interface AppConfig {
   website: string
   extraTab?: {
     webviewUrl: string
+    useEdgeProvider: boolean
```

Comment:
This naming conflicts with hooks naming convention `useXyz`. Perhaps to avoid confusion, could we name it `tabType: 'provider' | 'webview'`?

Reply:
will use `tabType: 'edgeProvider' | 'webview'`